### DISCUSSION TITLE
layout

### DISCUSSION THREAD

**Discussion by giscus** [Yiov/vitepress-doc#13] [General]
# layout

我的vitpress文档教程

https://vitepress.yiov.top/layout.html

**Comment by xkfe** [comment]
这个更新时间能不能自动根据 github 提交时间或者保存时间显示啊？每次修改这个 date 好麻烦

**Reply by Yiov** [reply]
可以的，不过官方显示在页面底部

https://vitepress.dev/zh/reference/default-theme-last-updated#last-updated

**Reply by xkfe** [reply]
如何使用您这个自定义顶部展示的样式，并使用官方的这个自动更新最后修改时间呢？

**Reply by Yiov** [reply]
插槽doc-before只能在标题上方，标题下方的话，或许自定义组件可以实现

**Reply by Yiov** [reply]
现在可以了，你可以试试看，[标题下添加时间](https://vitepress.yiov.top/components.html#%E6%A0%87%E9%A2%98%E4%B8%8B%E6%B7%BB%E5%8A%A0%E6%97%B6%E9%97%B4)

**Comment by pythonthink** [comment]
站长您好！我利用不蒜子插件吗，在底部设置本站访客数和本站总访问量，发现在服务器上两个数据同时变化，是不是只能挂载到github上才可以正确计算，还是说哪里需要有改动，感谢您的回复，期待您的回复！

**Reply by Yiov** [reply]
我看你仓库没什么问题，能显示就没什么问题，看你的值是不是写错了，一个是pv 一个是uv

**Reply by pythonthink** [reply]
我看了一下，我是直接copy的您的代码，并没有改动，在本地里面两个数据是不同的，我认为应该是没错的，然后我在githpage上好像是不同的数值，但是放到服务器上就是相同的了，也不知道是由于什么问题哈哈哈哈

**Reply by Yiov** [reply]
Ctrl+F5强制刷新缓存看看，如果还一样，配置也没问题，那就有可能是不蒜子偶尔的抽风

**Comment by LuckyJie12** [comment]
## 视图过度部分
切换深色模式的时候,屏幕会闪一下
